jenkins-builds/7.0.UP00-19.03-mc

```diff
@@ -34,6 +36,7 @@
  PyExtensions-2.7.15.6-CrayGNU-19.03.eb
  PyExtensions-3.6.5.6-CrayGNU-19.03.eb
  Python-bare-3.6.2.eb                               --hidden
+ Spark-2.3.1-CrayGNU-19.03-Hadoop-2.7.eb            --set-default-module
  VASP-5.4.4-CrayIntel-19.03.eb                      --set-default-module
  gperftools-2.7.eb                                  --set-default-module --installpath=$APPS/UES/jenkins/7.0.UP00/mc/easybuild/tools
```

Comment:
Should we put a comment here that we purposely have the tools at the end? Because everything else is alphabetically ordered.

Reply:
Yes, that is the silent agreement, However we cannot insert comments in the production list.